src/tools/cephfs/cephfs-shell

```diff
@@ -479,7 +479,16 @@ exists.')
         Create directory.
         """
         for dir_name in args.dirs:
-            path = to_bytes('/' + dir_name)
+            if dir_name[0] is '/':
+                path = to_bytes(dir_name)
+            else:
+                cwd = cephfs.getcwd().decode('utf-8')
+
+                if cwd is '/':
+                    path = to_bytes(cwd + dir_name)
+                else:
+                    path = to_bytes(cwd + '/' + dir_name)
```

Comment:
I don't think we should need to do any of this processing. `libcephfs` already has a concept of a current working directory and should handle all paths correctly. No preprocessing in `cephfs-shell` should be required.

The original code is wrong though. It should not be prepending a `/`. Can you remove that and give it another try? If that works, please remove the other instances of that if you find any.

Reply:
Thank you for explaining. Removing '/' works here. Few other places, where it has been used causes error if removed.